UI/window-basic-main.cpp

```diff
@@ -7868,18 +7869,59 @@ void OBSBasic::on_actionCopyFilters_triggered()
 	ui->actionPasteFilters->setEnabled(true);
 }
 
+bool OBSBasic::QueryPasteFilters(size_t count)
+{
+	QString text;
+	if (count == 1) {
+		text = QTStr("ConfirmPasteFilters.Text");
+	} else {
+		text = QTStr("ConfirmPasteFilters.TextMultiple")
+			       .arg(QString::number(count));
+	}
+
+	QMessageBox paste_filters(this);
```

Comment:
Today I was reminded of `OBSMessageBox::question` - it'd be worth using that here too.